### DISCUSSION TITLE
single PR for all incoming major versions for packages releasing majors frequently

### DISCUSSION THREAD

**Discussion by gberche-orange** [renovatebot/renovate#17710] [Request Help] [answered]
I have some packages that are publishing new major version every week even though they include no breaking changes, basically not following semVer.

Example https://github.com/pivotal-cf/cf-rabbitmq-smoke-tests-release/releases

To reduce noise triggered by having weekly major  PR being autoclosed and new one being created, I'd like a single PR "update package X to version Y" with the branch name "renovate/package-name", that would get updated for each version (both PR body and PR title)

I tried tuning the separateMajorMinor and [separateMultipleMajor](https://docs.renovatebot.com/configuration-options/#separatemultiplemajor) options but they don't seem aiming at this goal.

```
    {
      "description": "Prevent packages with many (e.g. weekly) major versions from creating noisy PRs, instead ask for a single PR that gets updated",
      "matchCurrentVersion": ">=100.0.0",
      "separateMajorMinor": false
    }
```

The [branchTopic](https://docs.renovatebot.com/configuration-options/#branchtopic) and [prTitle](https://docs.renovatebot.com/configuration-options/#prtitle) seem to low level.

Any suggestion of the right config to use for this use case ?

**Comment by PhilipAbed** [comment, marked as answer]
https://docs.renovatebot.com/noise-reduction/#package-grouping
is this what you are lookin for?

**Reply by rarkins** [reply]
Yeah, try giving it a groupName (even if same as the depName currently) so that it gets a `-major` suffix instead of the default `-{Major}.x`

**Reply by gberche-orange** [reply]
great, thanks @PhilipAbed and @rarkins !
